web/docs/data-model/operations/actions.md

```diff
@@ -247,6 +286,9 @@ const newTask = await createTask({ description: 'Keep learning TypeScript' })
 await markTasAsDone({ id: 1 })
 ```
 
+Wasp supports automatic full-stack type safety à la tRPC.
```

Comment:
> Wasp supports automatic full-stack type safety à la tRPC.
I'd avoid " à la" and "tRPC" 😂 Not because it's tRPC but rather a lot of people don't know it. Maybe we employ the "Show don't tell" principle somehow with a GIF or diagram showing what is "full-stack type safety"?

Reply:
Yeah, this is an old text written back when I thought everyone knew about tRPC.

I don't think I'll do a graph (I don't think it can show anything useful), but I'll change the wording.